### PR TITLE
Fix informing about curl error

### PR DESCRIPTION
**Pull request by alexby**
The exception about  curl error was never been thrown (we'r getting http_code exception before we are throwing curl error exception).

### CODE REVIEW

File: php/PaypalIPN.php [hunk 1]
```diff
@@ -121,19 +121,19 @@ public function verifyIPN()
         curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, 30);
         curl_setopt($ch, CURLOPT_HTTPHEADER, ['Connection: Close']);
         $res = curl_exec($ch);
-        $info = curl_getinfo($ch);
-        $http_code = $info['http_code'];
```
Comment (overint) [inline php/PaypalIPN.php]: Leave these lines here

File: php/PaypalIPN.php [hunk 2]
```diff
@@ -121,19 +121,19 @@ public function verifyIPN()
...
         if ( ! ($res)) {
             $errno = curl_errno($ch);
             $errstr = curl_error($ch);
             curl_close($ch);
             throw new Exception("cURL error: [$errno] $errstr");
         }
+
+        $info = curl_getinfo($ch);
+        $http_code = $info['http_code'];
+        if ($http_code != 200) {
```
Comment (overint) [inline php/PaypalIPN.php]: Just move the http code conditional below the curl error check.

**Comment by alexby** [comment]
I can't say I agree with you about that change. 
1) Code Complete teach us to define variables as far close as possible to the line where it will be used
2) $http_code is unused when $res is false

**Comment by overint** [comment]
That's fair enough. It seems you changed it, however I can't see the commit. 
If you can revert it to your original suggestion, i'll get it merged in.

**Comment by alexby** [comment]
Reverted to the initial commit

**Pull request merged**